impress-org/givewp-next-gen
# Fix: donation level input styles

## Description



At some point our branch merging duplicated the currency input file so there was an old implementation being used.  This resolved that issue which cleans up the strange looking donation level inputs from this:

<img width="287" alt="Screenshot 2023-04-04 at 5 06 55 PM" src="https://user-images.githubusercontent.com/10138447/229922556-247e2458-964d-4635-aef3-2900ef0cc8d9.png">

to this:

<img width="289" alt="Screenshot 2023-04-04 at 5 04 20 PM" src="https://user-images.githubusercontent.com/10138447/229922582-c1948c1a-d20c-46d6-8db1-8a2b50cc607b.png">


## Affects



The donation level inputs

## Visuals



See description

## Testing Instructions



- Run `npm run dev:form-builder`
- View the form builder donation levels and make sure they look right

## Pre-review Checklist
